packages/stable/src/api/geospatial/featureAPI.ts

```diff
@@ -190,4 +191,31 @@ export class FeatureAPI extends BaseResourceAPI<GeospatialFeatureResponse> {
       this.url(`${featureTypeExternalId}/features/aggregate`)
     );
   };
+
+  /**
+   * [List features](https://docs.cognite.com/api/v1/#tag/Geospatial/operation/listFeatures)
+   *
+   * ```js
+   * const searchParams = {
+   *  filter: {
+   *    and: [
+   *      { range:{ property: 'temperature', gt:4.54 } },
+   *      { stWithin: { property:'location', value:'POLYGON((60.547602 -5.423433, 60.547602 -6.474416, 60.585858 -5.423433, 60.547602 -5.423433))' } }
+   *   ]
+   *  },
+   * };
+   *
+   * const allFeaturesList = await client.geospatial.feature.list('ocean_temperature', searchParams);
+   * ```
+   */
+  public list = (
+    featureTypeExternalId: CogniteExternalId,
+    filterParams: GeospatialFeatureListFilter = {}
+  ) => {
+    const path = this.url(`${featureTypeExternalId}/features/list`);
+    return this.listEndpoint(
+      async (params) => this.post(path, { data: params }),
+      filterParams
+    );
+  };
```

Comment:
how do you know if it works? I suggest adding an integration test?
geospacial has 0 test coverage here, havind a few tests in the suite would be beneficial

Reply:
Oh! I think setting up a whole test suite will require more time.
I did this quickly after work due to [this](https://hub.cognite.com/ideas/list-functionality-for-js-sdk-featureapi-1202),
probably will have to ask for more time from my EM to work on this then.

Reply:
added test suite